src/main/java/frc/robot/subsystems/ESP.java

```diff
@@ -0,0 +1,30 @@
+// Copyright (c) FIRST and other WPILib contributors.
+// Open Source Software; you can modify and/or share it under the terms of
+// the WPILib BSD license file in the root directory of this project.
+
+package frc.robot.subsystems;
+
+import edu.wpi.first.wpilibj.DigitalOutput;
+import edu.wpi.first.wpilibj.smartdashboard.SmartDashboard;
+import edu.wpi.first.wpilibj2.command.SubsystemBase;
+import frc.robot.Constants;
+
+public class ESP extends SubsystemBase {
+  /** Creates a new ESP. */
+  DigitalOutput output; 
+  
+  public ESP() {
+    output = new DigitalOutput(Constants.LED_PORT); 
+  }
+
```

Comment:
please add some javadocs

Reply:
Added, Please review to ensure correct format.